ladybug_rhino/fromgeometry.py

```diff
@@ -141,8 +141,17 @@ def from_polyface3d(polyface):
 
 def from_face3d_to_wireframe(face):
     """Rhino PolyLineCurve from ladybug Face3D."""
-    return rg.PolylineCurve(
-        [from_point3d(pt) for pt in face.boundary] + [from_point3d(face.boundary[0])])
+    if face.has_holes == True:
```

Comment:
The PEP8 way to do it is to just say `if face.has_holes:`.

Here's the [official recommendation](https://www.python.org/dev/peps/pep-0008/#programming-recommendations):
```
Yes:   if greeting:
No:    if greeting == True:
Worse: if greeting is True:
```

Reply:
Complied